test/lib/random_schema.hh

```diff
@@ -20,12 +20,20 @@ class cql_test_env;
 namespace tests {
 
 class random_schema_specification {
+public:
+    using compress_sstable = bool_class<class compress_sstable_tag>;
+private:
     sstring _keyspace_name;
+    compress_sstable _compress;
 public:
-    explicit random_schema_specification(sstring keyspace_name) : _keyspace_name(std::move(keyspace_name)) { }
+    explicit random_schema_specification(sstring keyspace_name, compress_sstable compress = compress_sstable::yes)
+    : _keyspace_name(std::move(keyspace_name))
+    , _compress(compress)
```

Comment:
Again bad indentation.

Why add it to the constructor? Everything else is via the virtual functions?

Reply:
Since it won't change across invocations, I thought it should be treated the same as the keyspace name. But I synced with @denesb about this and I'll move it to the derived class (`default_random_schema_specification`).